VTEX - Ad Network API.json

```diff
@@ -3,7 +3,7 @@
     "info": {
         "version": "0.1.12",
         "title": "VTEX Ad Network API",
-        "description": "VTEX Ad Network connects VTEX stores with brands interested in advertising their products. This connection allows advertisers and merchants to boost their business:\r\n\r\n* **Advertisers:** Brands that can create ad campaigns to promote products in VTEX stores offering the same products in their catalogs, thus increasing sales. \r\n\r\n* **Merchants:** Can offer ad space in their stores for advertisers to promote products. In return, these merchants will receive a fee for each click on an ad in their stores.\r\n\r\n> Learn more about [VTEX Ad Network](https://help.vtex.com/en/tutorial/vtex-ad-network-beta--2cgqXcBuJmXN2livQvClur).\r\n\r\n## Index\r\n### Sponsored products\r\n `GET` [Get sponsored products for a query](https://developers.vtex.com/docs/api-reference/vtex-ad-network-api#get-/sponsored_products/-facets-)\r\n## Common parameters \r\n\r\n| **Parameter name** | **Description**   |\r\n| --------------- | ----------------- | \r\n| `{{accountName}}` | Name of the VTEX account. Used as part of the URL. |  \r\n| ` {{environment}}` | Environment to use. Used as part of the URL. |",
+        "description": "VTEX Ad Network connects VTEX stores with brands interested in advertising their products. This connection allows advertisers and publishers to boost their business:\r\n\r\n* **Advertisers:** Brands that can create ad campaigns to promote products in VTEX stores offering the same products in their catalogs, thus increasing sales. \r\n\r\n* **Publishers:** Can offer ad space in their stores for advertisers to promote products. In return, these publishers will receive a fee for each click on an ad in their stores.\r\n\r\n> Learn more about [VTEX Ad Network](https://help.vtex.com/en/tutorial/vtex-ad-network-beta--2cgqXcBuJmXN2livQvClur).\r\n\r\n## Index\r\n### Sponsored products\r\n `GET` [Get sponsored products for a query](https://developers.vtex.com/docs/api-reference/vtex-ad-network-api#get-/sponsored_products/-facets-)\r\n### Ad events\r\n `POST` [Send ad events](https://developers.vtex.com/docs/api-reference/vtex-ad-network-api#post-/ads)\r\n### Order events\r\n `POST` [Send order events](https://developers.vtex.com/docs/api-reference/vtex-ad-network-api#post-/order)\r\n### Banner\r\n `POST` [Get banner information](https://developers.vtex.com/docs/api-reference/vtex-ad-network-api#post-/banner/query)\r\n## Common parameters \r\n\r\n| **Parameter name** | **Description**   |\r\n| --------------- | ----------------- | \r\n| `{{accountName}}` | Name of the VTEX account. Used as part of the URL. |  \r\n| ` {{environment}}` | Environment to use. Used as part of the URL. |",
```

Comment:
Checking: Is it really not necessary to add common parameters such as the ones below?

`X-VTEX-API-AppKey`
`X-VTEX-API-AppToken`
`VtexIdclientAutCookie`

Reply:
Yes, these endpoints do not require authentication